pkg/cmd/server/api/v1/types.go

```diff
@@ -482,6 +482,8 @@ type IdentityProvider struct {
 	UseAsChallenger bool `json:"challenge"`
 	// UseAsLogin indicates whether to use this identity provider for unauthenticated browsers to login against
 	UseAsLogin bool `json:"login"`
+	// MappingMethod determines how identities from this provider are mapped to users
```

Comment:
I don't like duplicating the enumeration from identitymapper in server/api/types.go, but I also don't like code throughout referencing the server/api package...